modules/accredited_representative_portal/spec/requests/accredited_representative_portal/bypass_olive_branch_spec.rb

```diff
@@ -36,6 +36,16 @@ def normal = render json: {}
     end
   end
 
+  context 'when the request is for an accredited representative portal route staging' do
+    let(:path_prefix) { '//accredited_representative_portal' }
```

Comment:
Nice that we added this as a test case which I assume fails without the fix you've introduce in application code!

If we decide to add comments as a follow-on to this bugfix, we'd potentially want most of the commentary to be reiterated again here.